Generator/Assets/Flags.cs

```diff
@@ -601,13 +606,14 @@ public class Flags
             /* 9 */RandomizerSettings.mapAndCompassSettings == MapAndCompassSettings.Start_With,
             /* 10 */RandomizerSettings.castleRequirements == CastleRequirements.Open,
             /* 11 */RandomizerSettings.palaceRequirements == PalaceRequirements.Open,
-            /* 12 */RandomizerSettings.skipMinesEntrance,
+            /* 12 */RandomizerSettings.MinesEntrance == MinesEntrance.NoWrestling,
```

Comment:
I don't exactly understand how this works, but I wanted to check if this should be changed to:
```
/* 12 */RandomizerSettings.MinesEntrance != MinesEntrance.Closed,
```
Should this still be true when the value is Open in addition to NoWrestling?

Reply:
yeah but both use different flag so do this would be problematic

Reply:
Based on my testing, my suggested change appears to behave as expected. That is, the NoWrestling behavior is maintained even for Open. It logically makes sense based on what I think I am looking at, but Lunar should confirm.